api/src/db/DynamoBusinessesDataClient.ts

```diff
@@ -101,19 +94,16 @@ export const DynamoBusinessesDataClient = (
         businessId: businessId,
       },
     };
-    return db
-      .send(new GetCommand(params))
-
-      .then((result) => {
-        if (!result.Item) {
-          throw new Error("Not found");
-        }
-        return doMigration(result.Item.data);
-      });
+    return db.send(new GetCommand(params)).then((result) => {
+      if (!result.Item) {
+        throw new Error("Not found");
+      }
+      return result.Item.data;
+    });
   };
 
   const put = async (businessData: Business): Promise<Business> => {
-    const migratedData = migrateData(businessData);
+    const migratedData = businessData;
```

Comment:
pebble - since fn migrateData is removed, we should delete the variable name and reference businessData

Reply:
Good call out! I've updated the code to remove the businessData variable and reference it directly as needed, now that the migrateData function has been removed. Thanks for pointing that out!